src/app/components/Header/index.tsx

```diff
@@ -147,10 +147,7 @@ export function Header() {
       },
     },
     { to: '/liquidity', title: t(translations.mainMenu.liquidity) },
-    {
-      to: 'https://bitocracy.sovryn.app/stake',
-      title: t(translations.mainMenu.staking),
-    },
+    { to: '/stake', title: t(translations.mainMenu.staking) },
```

Comment:
I think it still should be under a dropdown group, but not sure if under bitocracy one, maybe under finance or somewhere in the portfolio itself?
@t-z-u